frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -201,27 +192,35 @@ describe('Kubevirt create VM using wizard', () => {
   it(
     'ID(CNV-2447) Multiple VMs created using "Cloned Disk" method from single source',
     async () => {
-      const clonedDiskProvisionConfig = provisionConfigs.get(ProvisionConfigName.DISK);
-      const vm1Config = vmConfig('vm1', testName, clonedDiskProvisionConfig);
-      const vm2Config = vmConfig('vm2', testName, clonedDiskProvisionConfig);
-      vm1Config.startOnCreation = false;
-      vm1Config.networkResources = [];
-
-      const vm1 = await wizard.createVirtualMachine(vm1Config);
+      const vm1 = new VMBuilder(getBasicVMBuilder())
+        .setProvisionSource(provisionSources.Disk)
+        .setWaitForImport(false)
+        .setStartOnCreation(false)
+        .generateName()
+        .setDisks([getDiskToCloneFrom()])
+        .generateName('vm1')
```

Comment:
called twice, plus in other cases as well.
maybe we should also remove all instances of generateName calls if we do it by default

Reply:
Removed all generateName instances